docs/ReleaseNotes.md

```diff
@@ -1,6 +1,17 @@
 Release Notes for Version 14
 ============================
 
+Build 013
+-------
+Published as version 14.8.0
+
+New Features:
+* Added Locale.isValid() method which returns true iff all of the given locale components (language,
```

Comment:
iff?

Reply:
"iff" means "if and only if": https://en.wiktionary.org/wiki/iff

Reply:
oh, I see.  it's really new to me :)